src/applications/disability-benefits/all-claims/pages/addDisabilities.js

```diff
@@ -38,7 +51,25 @@ export const uiSchema = {
         },
       ),
       'view:descriptionInfo': {
-        'ui:description': uiDescription,
+        'ui:description': descriptionInfo,
+      },
+    },
+  },
+  // This object only shows up when the user tries to continue without claiming either a rated or new condition
+  'view:newDisabilityErrors': {
+    'view:newOnlyAlert': {
+      'ui:description': newOnlyAlert,
+      'ui:options': {
+        hideIf: formData =>
+          !newConditionsOnly(formData) || hasClaimedConditions(formData),
+      },
+    },
+    'view:increaseAndNewAlert': {
+      'ui:description': increaseAndNewAlert,
+      'ui:options': {
+        hideIf: formData =>
+          // Only show this alert if the veteran is claiming both rated and new conditions
+          !newAndIncrease(formData) || hasClaimedConditions(formData),
```

Comment:
🤔 if we want this to show to veterans that *are* claiming 'new and increase', is `!newAndIncrease` what we want?